app/src/main/java/org/oppia/app/topic/revisioncard/RevisionCardViewModel.kt

```diff
@@ -28,7 +28,8 @@ class RevisionCardViewModel @Inject constructor(
 ) : ViewModel() {
   private lateinit var topicId: String
   private var subtopicId: Int = 0
-  private lateinit var binding: RevisionCardFragmentBinding
+  private lateinit var view: TextView
```

Comment:
Ditto for this view model (add TODO since it may be a memory leak).

Reply:
Done